Procfile.dev

```diff
@@ -1,3 +1,3 @@
 web: unset PORT && env RUBY_DEBUG_OPEN=true bin/rails server
 js: yarn dev
-sidekiq: bin/sidekiq
+sidekiq: bin/sidekiq
```

Comment:
Do we want to add it?

Reply:
I mean, I have it removed locally, but I think that's important when you run backend and frontend locally at the same time